Public/Src/FrontEnd/Nuget/NugetFrameworkMonikers.cs

```diff
@@ -111,65 +120,46 @@ public NugetFrameworkMonikers(StringTable stringTable)
+            NetStandard11 = Register(stringTable, "netstandard1.1", ".NETStandard1.1", ref NetCoreVersionHistory);
+            NetStandard12 = Register(stringTable, "netstandard1.2", ".NETStandard1.2", ref NetCoreVersionHistory);
+            NetStandard13 = Register(stringTable, "netstandard1.3", ".NETStandard1.3", ref NetCoreVersionHistory);
+            NetStandard14 = Register(stringTable, "netstandard1.4", ".NETStandard1.4", ref NetCoreVersionHistory);
+            NetStandard15 = Register(stringTable, "netstandard1.5", ".NETStandard1.5", ref NetCoreVersionHistory);
+            NetStandard16 = Register(stringTable, "netstandard1.6", ".NETStandard1.6", ref NetCoreVersionHistory);
+            NetStandard20 = Register(stringTable, "netstandard2.0", ".NETStandard2.0", ref NetCoreVersionHistory);
+            NetCoreApp20  = Register(stringTable, "netcoreapp2.0",  ".NETCoreApp2.0", ref NetCoreVersionHistory);
+            NetCoreApp21  = Register(stringTable, "netcoreapp2.1",  ".NETCoreApp2.1", ref NetCoreVersionHistory);
+            NetCoreApp22  = Register(stringTable, "netcoreapp2.2",  ".NETCoreApp2.2", ref NetCoreVersionHistory);
+            NetCoreApp30  = Register(stringTable, "netcoreapp3.0",  ".NETCoreApp3.0", ref NetCoreVersionHistory);
+            NetStandard21 = Register(stringTable, "netstandard2.1", ".NETStandard2.1", ref NetCoreVersionHistory);
+
+            Net10  = Register(stringTable, "net10",  ".NETFramework1.0", ref FullFrameworkVersionHistory);
+            Net11  = Register(stringTable, "net11",  ".NETFramework1.1", ref FullFrameworkVersionHistory);
+            Net20  = Register(stringTable, "net20",  ".NETFramework2.0", ref FullFrameworkVersionHistory);
+            Net35  = Register(stringTable, "net35",  ".NETFramework3.5", ref FullFrameworkVersionHistory);
+            Net40  = Register(stringTable, "net40",  ".NETFramework4.0", ref FullFrameworkVersionHistory);
+            Net45  = Register(stringTable, "net45",  ".NETFramework4.5", ref FullFrameworkVersionHistory);
+            Net451 = Register(stringTable, "net451", ".NETFramework4.5.1", ref FullFrameworkVersionHistory);
+            Net452 = Register(stringTable, "net452", ".NETFramework4.5.2", ref FullFrameworkVersionHistory);
+            Net46  = Register(stringTable, "net46",  ".NETFramework4.6", ref FullFrameworkVersionHistory);
+            Net461 = Register(stringTable, "net461", ".NETFramework4.6.1", ref FullFrameworkVersionHistory);
+            Net462 = Register(stringTable, "net462", ".NETFramework4.6.2", ref FullFrameworkVersionHistory);
+            Net472 = Register(stringTable, "net472", ".NETFramework4.7.2", ref FullFrameworkVersionHistory);
         }
 
-        private PathAtom Register(StringTable stringTable, string smallMoniker, string largeMoniker)
+        private PathAtom Register(StringTable stringTable, string smallMoniker, string largeMoniker, ref List<PathAtom> versions)

```

Comment:
You don't need `ref` here.  I kind of like that you did use `ref` because it draws attention to this list and makes it clear that it will be mutated; but on the other hand, it's not very idiomatic to pass a list by reference when you just want to add elements to the list.  I don't mind it either way.

Reply:
Removed it.